Resources/Prototypes/Traits/physical.yml

```diff
@@ -40,12 +40,12 @@
         - WillToLive
   components:
     - type: DeadModifier
-      deadThresholdModifier: -15
+      deadThresholdModifier: -10
```

Comment:
```suggestion
      deadThresholdModifier: -15
```

Reply:
So we keep Will to Live the same, but up the threshold of Will to Die and increase the points it gives you?

Reply:
I guess it's not a bad change, it's just weird from my perspective.